includes/compat/trait-wc-stripe-subscriptions.php

```diff
@@ -466,17 +466,13 @@ public function process_subscription_payment( $amount, $renewal_order, $retry =
 					}
 				}
 
-				$localized_messages = WC_Stripe_Helper::get_localized_messages();
-
-				if ( 'card_error' === $response->error->type ) {
-					$localized_message = isset( $localized_messages[ $response->error->code ] ) ? $localized_messages[ $response->error->code ] : $response->error->message;
-				} elseif ( 'payment_intent_mandate_invalid' === $response->error->type ) {
+				if ( 'payment_intent_mandate_invalid' === $response->error->type ) {
```

Comment:
Shall we include `payment_intent_mandate_invalid` in the `get_localized_messages`? As we have the following fallback in the `get_localized_error_message_from_response` helper method?

```
if ( isset( $localized_messages[ $error->type ] ) ) {
			return $localized_messages[ $error->type ];
		}
```